torchrl/_utils.py

```diff
@@ -252,36 +305,45 @@ def unsupported(*args, **kwargs):
         if func_name in implementations:
             try:
                 # check that backends don't conflict
-                module = import_module(self.module_name)
-                version = module.__version__
-
-                if (self.from_version is None or version >= self.from_version) and (
-                    self.to_version is None or version < self.to_version
-                ):
-                    warnings.warn(
-                        f"Got multiple backends for {func_name}. "
-                        f"Using the last queried ({module} with version {version})."
-                    )
-                else:
-                    return implementations[func_name]
+                version = self.import_module(self.module_name)
+                if self.check_version(version, self.from_version, self.to_version):
+                    if VERBOSE:
+                        warnings.warn(
+                            f"Got multiple backends for {func_name}. "
+                            f"Using the last queried ({module} with version {version})."
```

Comment:
This is the actual behaviour, in contrast with the docstring which is wrong.
I wonder if for now rather than VERBOSE we shouldn't be using RL_WARNINGS? Then mention in the warning that it can be turned off by setting RL_WARNINGS to False...

Reply:
Previously, there a was a comment saying "we don't use RL_WARNINGS because it's utterly verbose". It seems indeed that RL_WARNINGS is an environment variable set everywhere through the code. Why don't using the standard logging API?

Reply:
yeah we probably should :)